english/big-nosed-bugs/big-nosed-bugs.md

```diff
@@ -0,0 +1,8 @@
+You have arrived in a land full of ... well Big Nosed Bugs.
```

Comment:
Ah you are right! I will do that, thank you! 
 
 
 
 
-------Original Message-------
 
From: Casey Collaborator
Date: 7/8/2015 3:55:52 PM
To: udacity/create-your-own-adventure
Cc: gpickard
Subject: Re: [create-your-own-adventure] Add of big-nosed-bugs to story in 
english' directory (#1820)
 
In english/big-nosed-bugs/big-nosed-bugs.md:
> @@ -0,0 +1,8 @@
> +You have arrived in a land full of ... well Big Nosed Bugs.

It looks like this page is not reachable from the rest of the story. That is
 if a reader starts in language.md and keeps clicking links, they will never
find this page. You can fix this by finding a good place in the existing
story to add an option with a link that leads to this page.
—
Reply to this email directly or view it on GitHub.